lang/gcc10-bootstrap/files/patch-gcc10-i686-clang-bootstrap-fix.diff

```diff
@@ -0,0 +1,131 @@
+diff --git Makefile.in Makefile.in
```

Comment:
Looks like this patch is other version for https://github.com/gcc-mirror/gcc/commit/54258e22b0846aaa6bd3265f592feb161eecda75

Am I right @kencu / @iains ?

Reply:
> On the Tiger front....
> 
> Looking through the config.log, `--without-zstd` indeed is sidelining the zstd checks, so the "lucky guess" seems to be working. 

Hey, no fair! Not a lucky guess -- I found the commit that made it work.

https://github.com/gcc-mirror/gcc/commit/0fb0240a051df91d3c24385d1d3c17548b266544

Reply:
yes, but I am not sure that this patch is already back-ported in 10.3 .. it might be on the TODO for 10.4.  I'll try to check tomorrow.

This phase of the compiler devt. is a bit unfortunate - we have a lot of 'upstream' progress since 10.4 / 11.2 .. but there is still some time to go before 10.4 / 11.3 .. so the back port patch stack is quite large .. picking the critical pieces for 10.3 / 11.2 takes some time (and test hardware bandwidth) .. so patience might be needed :)

Reply:
@iains nevermind, I'm almost finished the dedicated branch that contains patches from 3 your branches for arm64, darwin and ppc that I've used here. Plus a few additional patches from this port.

As soon as I've finished tested it here, I'd love to send it to you to ask to add to TODO for 10.4 :) because you seems the right person.

Reply:
push it somewhere on github and I will try to do a comparison with my branches for 10.x, 11.x and master.